typings/index.d.ts

```diff
@@ -132,20 +133,20 @@ export interface PropGetters<Item> {
 }
 
 export interface Actions<Item> {
-  reset: (otherStateToSet?: {}, cb?: Callback) => void
+  reset: (otherStateToSet?: StateChangeOptions<Item>, cb?: Callback) => void
```

Comment:
Should it be `StateChangeOptions<Item>` or `Partial<StateChangeOptions<Item>>`?

Reply:
Good catch! Yes indeed it should be `Partial`